Assertion failed on opengl_interop.cpp

##### System information (version)
- OpenCV => 3.4.2
- Operating System / Platform => Windows 64 Bit
- Compiler => Visual Studio 2017

##### Detailed description
runtime error on [opengl_interop.cpp](https://github.com/opencv/opencv/blob/master/samples/opengl/opengl_interop.cpp)
`Exception: OpenCV(3.4.2-dev) d:\opencv\modules\imgproc\src\filterengine.hpp:363: error: (-215:Assertion failed) anchor.inside(Rect(0, 0, ksize.width, ksize.height)) in function 'cv::normalizeAnchor'`

See [filterengine.hpp:363](https://github.com/opencv/opencv/blob/3.4.2/modules/imgproc/src/filterengine.hpp#L363)

##### Steps to reproduce

compile and run [opengl_interop.cpp](https://github.com/opencv/opencv/blob/master/samples/opengl/opengl_interop.cpp)  (OpenCV library should be compiled with OpenGL support )

--- a/samples/opengl/opengl_interop.cpp
+++ b/samples/opengl/opengl_interop.cpp
@@ -360,7 +360,7 @@ protected:
         if (m_demo_processing)
         {
             // blur texture image with OpenCV on CPU
-            cv::blur(m, m, cv::Size(15, 15), cv::Point(-7, -7));
+            cv::blur(m, m, cv::Size(15, 15));
         }
 
         if (do_buffer)
@@ -385,7 +385,7 @@ protected:
         if (m_demo_processing)
         {
             // blur texture image with OpenCV on GPU with OpenCL
-            cv::blur(u, u, cv::Size(15, 15), cv::Point(-7, -7));
+            cv::blur(u, u, cv::Size(15, 15));
         }
 
         if (do_buffer)

